assembly/src/parsers/ast/context.rs

```diff
@@ -1,8 +1,8 @@
 use crate::ProcedureId;
 
```

Comment:
nit: let's remove a blank line after this import.

Reply:
fixed